Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11973,6 +11983,10 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets if the player can fly or not.",
 				},
+				SetCapabilities =
+				{
+					Notes = "Updates player's capabilities - flying, visibility, etc. from their gamemode.",
+				},
```

Comment:
Should we consider renaming the function before actually documenting it (and thus making it official API with a very bad name)?